elements/pfe-cta/README.md

```diff
@@ -31,7 +35,7 @@ _Note: `pfe-cta` is not a button, though it may look like one visually._
 
 ### Default slot
 
-We expect an anchor tag, `<a>` with an `href`, to be the first child inside `pfe-cta` element.
+We expect an anchor tag, `<a>` with an `href`, to be the first child inside `pfe-cta` element. Less preferred but allowed for specific use-cases include: `<button>` and `<input>`.
```

Comment:
@castastrophe   `<button>` & `<input>` elements inside a pfe-cta are not actually supported right now, in terms of styles. Should we remove this sentence for now?

![image](https://user-images.githubusercontent.com/456863/67222715-de188300-f3fb-11e9-85fa-d185d4408c5a.png)

![image](https://user-images.githubusercontent.com/456863/67222737-ea044500-f3fb-11e9-999a-12748270501d.png)

Reply:
Ah, I will update so that it clears button styles and we can fuss with input later. #becausewebstore we need buttons at least so I just pushed that up.